karate-junit5/src/main/java/com/intuit/karate/junit5/KarateFactory.java

```diff
@@ -0,0 +1,14 @@
+package com.intuit.karate.junit5;
+
+import org.junit.jupiter.api.TestFactory;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+@TestFactory
+public @interface KarateFactory {
```

Comment:
Does `@KarateTest` read even better?

Reply:
yes ! `@KarateTest` just merged before I saw this comment but you know what to do

Reply:
Can you please do the rename refactoring? Have to run...

Reply:
@sormuras will do, thanks !